bin/cron/export_mysql_database_to_redshift

```diff
@@ -29,6 +30,11 @@ def main
   end
   threads.each(&:join)
 
+  # The DMS tasks exported to staging (_import_) tables to avoid disrupting queries on the production Redshift tables
+  # during the lengthy table import process.  Rename the staging tables to the production table names to complete
+  # the import process
+  RedshiftImport.complete_table_import(Cdo::DMS.redshift_schemas_imported_from_database)
```

Comment:
Sounds like the rename process will take 5-10 minutes -- I wonder if you could explicitly give a time to run this final step (either via some logic here, or via separating it into a different cron), so we could be sure to move any Tableau refreshes out of that window?

Reply:
Each table rename takes less than a second, so I think the risk of disrupting the Tableau refreshes are minimal.  I'm also not certain we could easily coordinate two separate scheduled processes (one that executes the DMS exports and one that renames the tables, but only if the DMS tasks completed successfully).

Is there a way we can invoke the Tableau refreshes from this script as soon as the final import step completes (the table renaming)?

Reply:
Tableau refreshes are generally controlled via UI (and I'd like to keep it that way for ease of use).

Even if they're only unavailable for a second, what happens if some table goes missing mid-query? Seems easier to control if we just knew "these 5 minutes are a no-go for any processing".

Couldn't you just set it to wait to rename until some specified time?